code/game/objects/items/oddities.dm

```diff
@@ -153,18 +153,22 @@
 /obj/item/oddity/common/paper_omega
 	name = "collection of obscure reports"
 	desc = "Even the authors seem to be rather skeptical about their findings. The reports are not connected to each other, but their results are similar. The logo is faded, making it hard to tell if this was Greyson, Soteria, or an independent group."
-	icon_state = "paper_omega"
+	icon_state = "reports1"
 	prob_perk = 45 //The skeptical mind of the inker helps keep it grounded but it shows something unable to be shaken out of the mind
 	oddity_stats = list(
 		STAT_MEC = 6,
 		STAT_COG = 6,
 		STAT_BIO = 6
 	)
 
+/obj/item/oddity/common/paper_omega/Initialize()
+	icon_state = "reports[rand(1,7)]" // All these unused sprites will now add variety to reports. - Seb
```

Comment:
These are not unused sprites

Reply:
I've looked all over code and searched far and wide, nothing that spawns naturally ever uses those icons (theta, epsilon, nuke, etc)

Reply:
sec-docs do